CHANGES.rst

```diff
@@ -204,6 +218,8 @@ API changes
 - ``astropy.conftest.py``
 
 - ``astropy.constants``
+  - Changed constant names for solar/planetary conversion factors 
+    (R_sun -> R_Sun, R_earth -> Re_Earth, etc.)
```

Comment:
A big :-1: for this name change from me. 
 * They need to be at least deprecated first before removal. 
 * doing such an API change just to change the case is totally unneccessary, and leaves inconsistency in with the rest of the code.
 * I'm pretty sure most of us working with planets want to keep using R_earth and R_jup as is, without the extra complications of whether it is polar or equatorial.